controller/space.go

```diff
@@ -358,8 +361,10 @@ func ConvertSpaceFromModel(request *goa.RequestData, p space.Space, additional .
 			Version:     &p.Version,
 		},
 		Links: &app.GenericLinksForSpace{
-			Self:    &selfURL,
-			Backlog: &relatedBacklogList,
+			Self:              &selfURL,
+			Backlog:           &relatedBacklogList,
```

Comment:
Done in https://github.com/almighty/almighty-core/pull/1033/commits/5ff16ae06bb05afc3c799e508e50ebf17fece5aa
thanks for heads up !